intro-to-git/commits.md

```diff
@@ -0,0 +1,290 @@
+# Commits
+
+## Learning Goals
+
+- Explain how a commit in Git is made using the git add and git commit commands
+- Practice versioning code by creating commits in Git
+- Practice observing commit history using the commands git status, git log, git show, and git diff
+- Explain different ways of "undoing" commits using git revert and git reset
+
+## Introduction
+
+We know that Git will help us do these two things:
+
+- Version our software with commits and look at our commit history
+- Collaborate with others
+
+The foundation of Git is understanding commits. A solid understanding of commits allows programmers to see their project's progress.
+
+Sometimes, we work on a project alone. Git is still an incredibly powerful and helpful tool, even without the collaboration!
+
+## Vocabulary and Synonyms
+
+| Vocab | Definition | Synonyms | How to Use in a Sentence |
+| ----- | ---------- | -------- | ------------------------ |
+
+Code Change
+Commit
+Local Changes Area
+Staging Area
+Untracked Area
+Tracking
+vim
+
+## Format
+
+The goal of this lesson is to learn how to build commits in Git. However, there is plenty of information to consider before that.
+
+This lesson is organized into three sections: We'll learn about...
+
+1. more details about what commits are
+1. the overall process of making a commit
+1. the steps to take to make a commit
+
+## How Do We See Differences in Code?
+
+One piece of information that commits contain is the differences (diff) in code between this version, and the last version.
+
+We can generalize that each change in software is a set of files and...
+
+- The lines of code that were added; what they are, what file, and what line number
+- The lines of code that were deleted
+- The lines of code that were modified
```

Comment:
I personally (also?) learned that a modified line is a deletion and an addition, but I feel like I've gotten pushback in the classroom for a preference to call it modify.

I'll think on this!!! I'll probably go with adding this parenthetical ;) because it is way cooler (and very helpful) to think of diffs as only added + deleted